e2e/app.e2e.spec.ts

```diff
@@ -36,17 +39,17 @@ describe('iam-cache-server E2E tests', () => {
 
     didRegistry = await loadFixture(deployDidRegistry);
     process.env.DID_REGISTRY_ADDRESS = didRegistry.address;
-    process.env.IPFS_CLUSTER_ROOT = 'http://localhost:8080';
-    process.env.IPFS_CLUSTER_USER = 'not-required-locally';
-    process.env.IPFS_CLUSTER_PASSWORD = 'not-required-locally';
```

Comment:
Aren't the Cluster User and Password need to authenticate to the cluster when not running locally?

Reply:
No, they not needed. Initially this variables were required for environment validation. I made them optional and we don't have to specify empty strings now